00-HTML-CSS-basics/css 2.5/mock.html

```diff
@@ -0,0 +1,91 @@
+<!doctype html>
+<html class="no-js" lang="">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+
+        <link rel="stylesheet" href="mock.css">
+    </head>
+    <body size="cover">
+        <div class="container" align="center">
+            <header id="header" class="myThings">Buenos Aires</header>
+            <br>
```

Comment:
we shouldn't use `<br>` as this is only for styling purpose, instead you can use a div element with a class, and from css you can set the height it should fill in order to create a blank space.. or better, add a margin bottom to your header